README.md

```diff
@@ -408,6 +408,51 @@ const result = validate(value, validations)
 } */
 ```
 
+### Custom functions
+Execute custom functions to specific validations cases
+
+`propName` (string) - Name of the validation rule returned on error
+
+`validation` (function: boolean) - Function that will be called with value argument
+
+```javascript
+const cardNumber = "1234";
+
+// Single Validation
+const propName = "invalidCardNumber";
+const validation = (value) => value.length === 16;
+const validations = {
+    custom: {
+        [propName]: validation,
```

Comment:
Hello, Jhow!

Why is this example used with `[propName]` and an example below with multiple validations is used with functions like` invalidCardNumber: (value) => value.length === 16`?